include/swift/Serialization/ModuleFormat.h

```diff
@@ -755,7 +755,7 @@ namespace decls_block {
     IdentifierIDField, // name
     DeclContextIDField,// context decl
     TypeIDField, // underlying type
-    TypeIDField, // interface type
+    TypeIDField, // interface type (no longer used)
```

Comment:
As the designer of the format, I need to exercise its forward-compatibility support every now and then to remind myself it exists. Also, it's a mild nicety for people who only rebuild their compiler.